pkg/logs/client/tcp/destination.go

```diff
@@ -27,6 +28,8 @@ type Destination struct {
 	connManager         *ConnectionManager
 	destinationsContext *client.DestinationsContext
 	conn                net.Conn
+	connResetInterval   time.Duration
```

Comment:
This interval is available through the `connManager` endpoint object, that may not be worth duplicating the info here?
At some point, I would even consider having a method in the `ConnectionManager` like `ShouldReset()`?

Reply:
Makes sense, let me make that change